torcx/what-is-torcx.md

```diff
@@ -0,0 +1,31 @@
+# What is Torcx?
+
+[Torcx][gh-torcx] is a new boot-time addon manager designed specifically for Container Linux. At the most basic level, it is a tool for applying ephemeral changes to an immutable system during early boot. This includes providing third-party binary addons and installing systemd units, which can vary across environments and boots. On every boot, Torcx reads its configuration from local disk and propagates specific assets provided by addon packages (which must be available in local stores).
+
+## Torcx overview
+
+Torcx is meant to be a complementary tool to [Ignition][ignition] and [systemd][systemd], customizing a Container Linux system at runtime without having to build a custom OS image. This goal is achieved by following two main principles: customizations are ephemeral and they are applied exactly once per-boot. Torcx also has a very simple design, with the aim of providing a small low-level system utility which can be driven by more advanced and higher-level tools.
```

Comment:
I double-checked as I was surprised by the verb, and I think you meant "complements". Otherwise, applied as-is.